nemo/collections/nlp/modules/common/transformer/transformer.py

```diff
@@ -84,11 +85,13 @@ def __init__(
         hidden_act: str = 'relu',
         mask_future: bool = False,
         pre_ln: bool = False,
+        pre_ln_final_layer_norm: bool = True,
```

Comment:
Default should be False in order to be able to load the old checkpoints. When we load a model from nemo file, the config file is read form there. You may add this param to the latest config files with value of True so that new models get trained with last layer norm enbaled.

Reply:
I think the default should be pre_ln_final_layer_norm=True because it's the correct way. sandeep will send a patch for NMT setting it to False if the attribute is not in the config

Reply:
I see, then it is fine if Sandeep plan to fix is with another patch. Just a heads up that the current code can not load previous checkpoints which do not have this last layer norm.

Reply:
please keep default to True

Reply:
@okuchaiev  Yes. Sandeep has patched it in this PR.